README.md

```diff
@@ -1 +1,203 @@
-# leaderboard-API
\ No newline at end of file
+<a name="readme-top">LEADERBOARD</a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [LEADERBOARD] <a name="about-project"></a>
+
+**[Leaderboard]** is a project to display diffrent scores of participants in a contest.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://nodejs.org/">Node js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>API</summary>
+  <ul>
+    <li><a href="https://www.postman.com/">Postman</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **[The application has form inputs to collect score data and display it on a scoreboard]**
+- **[the refresh button checks for new entries in the Api and displays it on the screen]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link]()

```

Comment:
- [ ] Kindly don't leave the live link blank. Either delete the section or indicate that's under construction for a professional README file. 🙏